python/cugraph/cugraph/tests/link_prediction/test_overlap_mg.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2022, NVIDIA CORPORATION.
+# Copyright (c) 2023, NVIDIA CORPORATION.
```

Comment:
same as above, this should be a range now (2022-2023)

Reply:
fixed